app/src/main/java/de/westnordost/streetcomplete/quests/QuestsModule.kt

```diff
@@ -378,12 +378,12 @@ whether the postbox is still there in countries in which it is enabled */
 
     // roads
     AddSidewalk(), // for any pedestrian routers, needs minimal thinking
+    AddShoulder(), // needs minimal thinking
     AddRoadSurface(), // used by BRouter, OsmAnd, OSRM, graphhopper, HOT map style... - sometimes requires way to be split
     AddTracktype(), // widely used in map rendering - OSM Carto, OsmAnd...
     AddCycleway(countryInfos), // for any cyclist routers (and cyclist maps)
     AddLanes(), // abstreet, certainly most routing engines - often requires way to be split
     // AddStreetParking(),
```

Comment:
> Alternatively this if it's down there because it's influenced by data from AddCycleway/AddLanes/AddStreetParking. 😄

It is?

Reply:
But yeah, it should be after AddStreetParking because a parking lane can be/look very similar to a shoulder.

Reply:
> It is?

It looks like it to me from the code:
https://github.com/streetcomplete/StreetComplete/blob/efd54939e573836d7ed5a9c224ee84192b690b94/app/src/main/java/de/westnordost/streetcomplete/quests/shoulder/AddShoulder.kt#L36-L44

Reply:
> But yeah, it should be after AddStreetParking because a parking lane can be/look very similar to a shoulder.

Okay, then:
```suggestion
    AddRoadSurface(), // used by BRouter, OsmAnd, OSRM, graphhopper, HOT map style... - sometimes requires way to be split
    AddTracktype(), // widely used in map rendering - OSM Carto, OsmAnd...
    AddCycleway(countryInfos), // for any cyclist routers (and cyclist maps)
    AddLanes(), // abstreet, certainly most routing engines - often requires way to be split
    // AddStreetParking(),
    AddShoulder(), // needs minimal thinking, but after AddStreetParking because a parking lane can be/look very similar to a shoulder
```